index.html

```diff
@@ -9025,6 +9039,267 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
 </section> <!-- /Take Element Screenshot -->
 </section> <!-- /Screen capture -->
 
+<section>
+<h2>Print</h2>
+
+<p>The print functions are a mechanism to render the document to a
+ paginated format. It is returned to the <a>local end</a> as a Base64
+ encoded string containing a PDF representation of the paginated
+ document.
+
+<p class=note>
+For historical reasons, all dimensions relating to the printed output
+are specified in inches.
+
+<p>When required to <dfn>parse a page range</dfn> with
```

Comment:
How does the local end tell the valid values for the page range? I can't see that in this diff, but maybe I've missed it.

Reply:
That isn't covered; I don't know what a solution would look like. In practice I'm guessing this range stuff is relatively uncommon and people mostly just want the full output.

Reply:
It's in the line [9233](https://github.com/w3c/webdriver/blob/16fd8ee87ddac27cad9000eae5129a405496dc0e/index.html#L9233). That's it?